src/platforms/ctxlink/README.md

```diff
@@ -0,0 +1,49 @@
+# Firmware BMP for ctxLink Wireless Probe
+
+Allows the use of ctxLink board as a Black Magic Probe.
+
+## LED assignments
+ * PB2 -> LED0 -> Green - Running LED
+ * PC6 -> LED1 -> Orange - Idle LED
+ * PC8 -> LED2 -> Red - Error
+ * PC9 -> LED3 -> Green - ctxLink Mode
+
+## Connections
+
+* PA3: TDI
+* PA4: TMS/SWDIO
+* PA5: TCK/SWCLK
+* PC7: TDO/TRACESWO
+* PA2: TRST
+* PA2: nRST
```

Comment:
This lists PA2 twice, is that correct or a typo? (especially given nRST is not nTRST)